content/applications/inventory_and_mrp/purchase/products/temporary_reordering.rst

```diff
@@ -0,0 +1,121 @@
+Temporary reordering rules
+==========================
+
+Some businesses require certain products to always have a minimum quantity of stock on-hand at any
+given time. To avoid stock falling below a certain threshold, companies can create *reordering
+rules* in Odoo to automate purchase orders for specific products.
+
+Reordering rules keep the forecasted stock levels above a certain threshold without exceeding a
+specified upper limit. When a product with a reordering rule falls below a specified quantity, Odoo
+generates an order using the specified *route* (e.g. *Buy* or *Manufacture*) to replenish the stock.
+
+In certain cases, businesses might opt for *temporary reordering rules* when they do not want
+specific products to be replenished automatically.
+
+In Odoo, a "temporary" reordering rule is created in the replenishment dashboard when a product:
+
+#. with a *Buy* route
+#. has no reordering rule configured
+#. has 0 quantity in stock
+#. is included in a sales order (SO).
+
+This rule is deleted upon confirmation of the purchase order (PO) generated for the product.
+
+.. seealso::
+   - :doc:`../../inventory/product_management/product_replenishment/reordering_rules`
+   - :doc:`../../purchase/products/reordering`
+
+Configuration
+=============
+
+To configure a product that triggers temporary reordering rules when its stock reaches 0, begin by
+going to :menuselection:`Inventory app --> Products --> Products`, and click :guilabel:`New`.
+
+On the product form, enter the product name, and ensure the :guilabel:`Can be Sold` and
+:guilabel:`Can be Purchased` options are enabled underneath the :guilabel:`Product Name` field.
+Finally, set the :guilabel:`Product Type` to `Storable Product`, under the :guilabel:`General
+Information` tab.
+
+Next, click the :guilabel:`Purchase` tab. Under :guilabel:`Vendor`, click :guilabel:`Add a line`,
+select a vendor from the drop-down menu, and set a purchase price under :guilabel:`Price`.
+
+.. important::
+   A vendor **must** be set for temporary reordering rules to work. Although a :abbr:`PO (purchase
+   order)` will still be created automatically, attempting to replenish the product from the
+   :guilabel:`Replenishment` dashboard in the *Inventory* app triggers a warning to add a vendor on
+   the product form.
+
+   .. image:: temporary_reordering/temporary-reordering-warning-popup.png
+      :align: center
+      :alt: Warning pop-up upon clicking to replenish product with no set vendor.
+
+Before creating a :abbr:`SO (sales order)` for the product, ensure that the :guilabel:`On Hand`
+smart button on the product form reads `0.00 Units`. Then, ensure that the :guilabel:`Reordering
+Rules` smart button reads `0`, meaning there are no rules applied to this product.
+
+.. image:: temporary_reordering/temporary-reordering-smart-buttons.png
+   :align: center
+   :alt: Product form smart button row displaying reordering rules and on hand buttons.
+
+Create sales order
+==================
```

Comment:
May want to consider re-titling this: "Trigger temporary reordering rule" (or something like that)